wordpress-mobile/WordPress-iOS
# [Insights Management] Save settings for each site

Fixes #12249 

This change saves Insights settings for each site separately.

- Initially displays these stats in this order: 
  - _Posting Activity_ 
  - _Today_ 
  - _All-Time_ 
  - _Most Popular Time_ 
  - _Comments_
- Saves and loads Insights settings per site.
- Updates when a new Insight is added from 'Add New Stats Card'.

To test:

---
Initial Stats displayed:
- Start with a fresh install.
- Run the app, access Stats > Insights.
- Verify only these are displayed, in this order:
  - Customize card
  - Posting Activity
  - Today
  - All Time
  - Most Popular
  - Comments
  - Add stat card

---
Add Stat card:
- Select 'Add new stats card'.
- Select a stat that is not already displayed. (Note there is no indication that a stat is already being displayed. That is coming soon.)
- Verify the selected stat appears at the bottom of the Insights table, but before the 'Add new stats card' card.
- Switch sites.
- Verify the stat added above does not appear for this site.

---
Customize card:
- Dismiss the 'Customize your insights' card.
- Verify it no longer appears for _any_ site.

---
Update release notes:
